apps/condo/domains/common/components/TextArea.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from '@emotion/styled'
+import { Input  } from 'antd'
+import { TextAreaProps } from 'antd/es/input'
+import React from 'react'
+
+import { colors } from '@open-condo/ui/dist/colors'
+
+
+const StyledTextArea = styled(Input.TextArea)`
```

Comment:
Hey, what is our mainstream way of handling styles in the project? I thought we decided to ditch styled components...